README.md

```diff
@@ -41,10 +41,10 @@ installers for all supported operating systems.
 
 #### Debian and Ubuntu based Package Repository (GNU/Linux x86/x64)
 
-1. Save the following as `/etc/apt/sources.list.d/etcher.list`:
+1. Add Etcher debian reposirtory:
 
     ```
-    deb https://dl.bintray.com/resin-io/debian stable etcher
+    echo "deb https://dl.bintray.com/resin-io/debian stable etcher" | sudo tee /etc/apt/sources.list.d/etcher.list
```

Comment:
@jviotti Like the commands below this (`sudo apt-get` ...), they assume the developer is not running as root. A redirect of `stdout` would fail since non-root users don't have write access to `/etc`.

Let me know if I should add these details to the commit log message.